cpu/stm32_common/periph/eth.c

```diff
@@ -226,7 +226,7 @@ int stm32_eth_init(void)
       stm32_eth_set_mac(eth_config.mac);
     }
     else {
-      luid_get(hwaddr, ETHERNET_ADDR_LEN);
+      luid_get_eui48(hwaddr);
```

Comment:
> `expected 'eui48_t * {aka struct <anonymous> *}' but argument is of type 'char *'`

Reply:
sorry: i overlooked a warning since my external module forced me to not promote them